package.json

```diff
@@ -25,17 +28,20 @@
     "index.js"
   ],
   "license": "MIT",
+  "dependencies": {
+    "ember-cli-babel": "^5.1.5"
+  },
   "devDependencies": {
     "broccoli-asset-rev": "^2.4.3",
-    "coveralls": "^2.11.13",
-    "ember-cli": "2.9.1",
+    "coveralls": "^2.11.15",
+    "ember-cli": "^2.10.0",
```

Comment:
Sure, I'll re-pin it to the exact version. 

This really highlights how the CLI doesn't quite fit with Semver. A minor version *should not require changes* but always does.